src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -100,12 +101,19 @@ public void save(AddressBook addressBook) throws StorageOperationException {
         try (final Writer fileWriter =
                      new BufferedWriter(new FileWriter(path.toFile()))) {
 
+
+
+
```

Comment:
Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.